zfm.zsh

```diff
@@ -118,14 +118,42 @@ Default Keybindings:
 
 Ctrl+P                                  Select a bookmarked directory and jump to it
 Ctrl+O                                  Select one or multiple bookmarks and insert them into the current command line
+
+ENV Configuration:
+ZFM_BOOKMARKS_FILE                      Setting this allows the user to choose where the ZFM_BOOKMARKS_FILE should be located. e.g.: export ZFM_BOOKMARKS_FILE=\"$HOME\.dots\zmf.txt\"
 "
 
-function zfm()
+# CHANGE: Added logic to it's own funnction
+function set_bookmarks_file() 
 {
-    local bookmarks_file="${HOME}/.zfm.txt"
+    # CHANGE: If ZFM_BOOKMARKS_FILE is set then use that, otherwise fall back to the default location
+    [[ $ZFM_BOOKMARKS_FILE ]] && local bookmarks_file="$ZFM_BOOKMARKS_FILE" || local bookmarks_file="${HOME}/.zfm.txt"
```

Comment:
The correct POSIX compliant way of checking that a variable is set would be to do [[ -n "{-z ${ZFM_BOOKMARKS_FILE+x}" ]], see https://stackoverflow.com/questions/42655304/how-do-i-check-if-a-variable-is-set-in-zsh/42655305

Reply:
Thanks for that, updated.